apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts

```diff
@@ -89,8 +89,24 @@ export class CreateNovuIntegrations {
     }
   }
 
+  private async createInAppIntegration(command: CreateNovuIntegrationsCommand) {
+    await this.createIntegration.execute(
+      CreateIntegrationCommand.create({
+        providerId: InAppProviderIdEnum.Novu,
+        channel: ChannelTypeEnum.IN_APP,
+        name: 'Novu In-App',
+        active: true,
+        check: false,
+        userId: command.userId,
+        environmentId: command.environmentId,
+        organizationId: command.organizationId,
+      })
+    );
+  }
+
```

Comment:
Let's also add a count check to be extra sure there is no existing in app integration (only one is allowed per environment)

Reply:
Good call Gali, we need to ensure one in-app integration.